app/scripts/dateformat.js

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
You'll have to demonstrate how this bower dependency works, I havent been able to get it to work.